assets/css/admin.rtl.css

```diff
@@ -155,11 +155,93 @@
 	padding-top: 2px;
 }
 
+/* WCPay Fraud Risk Level meta box */
```

Comment:
@dmallory42 I wonder if these changed would be better to ship separately? they seem not related to the PR and there are no instructions where to test these design changes

Reply:
I have no idea where these changes came from. 😅 Will try and remove them from the PR!